straxen/analyses/posrec_comparson.py

```diff
@@ -0,0 +1,89 @@
+def load_corrected_positions(context, run_id, events,
+                             cmt_version='ONLINE',
+                             posrec_algos=['mlp', 'gcn', 'cnn']):
+
+    """
+    Returns the corrected position for each position algorithm available, without the need to reprocess
+    event_basics, as the needed information is already stored in event_posrec_many.
+    
+    :param cmt_version: CMT version to use (it can be a list of same length as posrec_algos, if different versions
+                        are required for different posrec algorithms, default 'ONLINE')
+    :param posrec_algos: list of position reconstruction algorithms to use (default ['mlp', 'gcn', 'cnn'])
+    """
+    
+    posrec_algos = (posrec_algos, ) if isinstance(posrec_algos, str) else posrec_algos
+
+    if hasattr(cmt_version, '__len__') and not isinstance(cmt_version, str) and len(cmt_version) != len(posrec_algos):
+        raise TypeError(f"cmt_version is a list but does not match the posrec_algos ({posrec_algos}) length.")
+        
+    cmt_version = [cmt_version] * len(posrec_algos) if isinstance(cmt_version, str) else cmt_version
+    
+    dtype = []
+    posrec_dependent_dtype = ['x', 'y', 'z', 'r', 'r_naive',
+                              'r_field_distortion_correction', 'theta']
+    
+    for algo in posrec_algos:
+
+        dtype += [
+                ((f'Interaction x-position, field-distortion corrected (cm) - {algo.upper()} posrec algorithm', f'x_{algo}'), 
+                 np.float32),
+                ((f'Interaction y-position, field-distortion corrected (cm) - {algo.upper()} posrec algorithm', f'y_{algo}'), 
+                 np.float32),
+                ((f'Interaction z-position, field-distortion corrected (cm) - {algo.upper()} posrec algorithm', f'z_{algo}'), 
+                 np.float32),
+                ((f'Interaction radial position, field-distortion corrected (cm) - {algo.upper()} posrec algorithm', f'r_{algo}'),
+                 np.float32),
+                ((f'Interaction r-position using observed S2 positions directly (cm) - {algo.upper()} posrec algorithm', f'r_naive_{algo}'),
+                 np.float32),
+                ((f'Correction added to r_naive for field distortion (cm) - {algo.upper()} posrec algorithm',
+                  f'r_field_distortion_correction_{algo}'), np.float32),
+                ((f'Interaction angular position (radians) - {algo.upper()} posrec algorithm', f'theta_{algo}'),
+                 np.float32)]
+        
+    dtype += [(('Interaction z-position using mean drift velocity only (cm)', 'z_naive'), np.float32)]
+    result = np.zeros(len(events), dtype=dtype)
+
+    drift_speed = context.show_config('event_positions', 'electron_drift_velocity').current.values[0]
+    drift_speed = drift_speed if drift_speed != strax.OMITTED \
+        else context.show_config('event_positions', 'electron_drift_velocity').default.values[0]
```

Comment:
For now, this seems like the reasonable approach but I would guess we'd also track this in CMT rather soon, right @ahiguera-mx ?

By the way, you can also do a little more simple command of:
```python
context.config.get('electron_drift_velocity')
```

Reply:
Then for the moment I will leave it like that and as soon as this is implemented in CMT I will change these lines (anyway, also event_positions will have to be changed)

Reply:
@jorana @ftoschi  regarding the `electron_drift_velocity` I thought that the implementation would use `drift_time`, I guess we decided to use `electron_drift_velocity` at the end. In any case we need to coordinate with ACs and the analyst to provide the data to add this into CMT

Reply:
The map itself is defined using the drift_time, but then when interpolating the map it is necessary to have identical units in all dimensions and for this reason we should go from drift time to z.